packages/teams-js/src/public/appInitialization.ts

```diff
@@ -58,12 +65,12 @@ export function notifyAppLoaded(): void {
 
 /**
  * @deprecated
- * As of TeamsJS v2.0.0, please use {@link app.notifySuccess app.notifySuccess(): void} instead.
+ * As of TeamsJS v2.0.0, please use {@link app.notifySuccess app.notifySuccess(): Promise<NotifySuccessResponse>} instead.
  *
  * Notifies the frame that app initialization is successful and is ready for user interaction.
  */
-export function notifySuccess(): void {
-  notifySuccessHelper(
+export function notifySuccess(): Promise<NotifySuccessResponse> {
```

Comment:
we shouldn't change the signature on this function because it's an old back compat function.

Reply:
Ohh got it, that makes sense. Will revert the change.